pipeline/vars/common.groovy

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env groovy
+/*
+    Common groovy methods that can be reused by the pipeline jobs.
+*/
+
+import groovy.json.JsonSlurper
+
+String composeId = ""
+String instanceName = ""
+
+def prepareNode() {
+    /*
+        Installs the required packages needed by the Jenkins node to
+        run and execute the cephci test suites.
+    */
+    sh '''
+        #!/bin/bash
+
+        echo "Initialize Node"
+        sudo yum install -y wget git-core python3
+
+        # Workaround: Disable IPv6 to have quicker downloads
+        sudo sysctl -w net.ipv6.conf.eth0.disable_ipv6=1
+
+        # Mount reesi for storing logs
+        if [ ! -d "/ceph" ]; then
+            echo "Mounting ressi004"
+            sudo mkdir -p /ceph
+            sudo mount -t nfs -o sec=sys,nfsvers=4.1 reesi004.ceph.redhat.com:/ /ceph
+        fi
+
+        # Copy the auth files from internal server
+        wget http://magna002.ceph.redhat.com/cephci-jenkins/.osp-cred-ci-2.yaml -O ${HOME}/osp-cred-ci-2.yaml
+        wget http://magna002.ceph.redhat.com/cephci-jenkins/.cephci.yaml -O ${HOME}/.cephci.yaml
+
+        # Install cephci pre-requisistes
+        rm -rf .venv
+        python3 -m venv .venv
+        source .venv/bin/activate
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt
+        deactivate
```

Comment:
is it possible to put this as `bootstrap.sh` and call the file here instead of writing shell script code here

Reply:
As the code was specific to `centos-7` template, I didn't want to create a separate file. However, I could keep it out in a separate file but locally to the `vars` folder.

Reply:
Thanks, taken care